web/client/src/components/form/TokenDisplay.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Typography, Card, CardContent, Button } from "@material-ui/core";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import PropTypes from "prop-types";
+/**
+ *
+ * @param {*} param0 token being passed for display
+ * @return {div} returns a card so the token stands
+ */
+export default function TokenDisplay({ token }) {
+  const [copy, setCopy] = React.useState(false);
+  return (
+    <div>
+      <Card>
+        <CardContent>
```

Comment:
No need to use `Card` or `CardContent` since the `MaterialUI` component adds a card background automatically to its children.